tests/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/terraform-ibm-modules/terraform-ibm-scc-da
```

Comment:
@ctolon22 Can you change this back?

Reply:
sorry, i had a different name in the fork producing this change in the pre-commit hook and missed it in the commit. Hope is ok now